README.md

```diff
@@ -1,46 +1,64 @@
-# devcontainers/ci
+# Dev Container Build and Run (devcontainers/ci)
 
-devcontainers/ci contains a GitHub action and Azure DevOps task aimed at making it easier to re-use a [development container](https://github.com/devcontainers/spec) in a GitHub workflow or Azure DevOps pipeline.
+The Dev Container Build and Run GitHub Action is aimed at making it easier to re-use [Dev Containers](https://containers.dev) in a GitHub workflow. The Action supports both using a Dev Container to run commands for CI, testing, and more along with pre-building Dev Container image. Dev Container image building supports [Dev Container Features](https://containers.dev/implementors/features/#devcontainer-json-properties) and automatically places Dev Container [metadata on an image](https://containers.dev/implementors/spec/#image-metadata) label for simplified use.
 
-This project builds on top of [@devcontainers/cli](https://www.npmjs.com/package/@devcontainers/cli)
+An similar [Azure DevOps Task](./docs/azure-devops-task.md) is also available!
 
+Note that this project builds on top of [@devcontainers/cli](https://www.npmjs.com/package/@devcontainers/cli) which can be used in other automation systems.
 
 ## GitHub Action
+The examples below shows usage of the GitHub Action - see the [GitHub Action documentation](./docs/github-action.md) for more details.
 
-The example below shows usage of the GitHub Action - see the [GitHub Action documentation](./docs/github-action.md) for more details:
+> **NOTE:** This Action is not currently capable of taking advantage of pre-built Codespaces. However, pre-built images are supported.
 
 
+**Pre-building an image:**
+
 ```yaml
-- name: Build and run dev container task
+- name: Pre-build dev container image
   uses: devcontainers/ci@v0.2
   with:
     imageName: ghcr.io/example/example-devcontainer
-    runCmd: make ci-build
+    cacheFrom: ghcr.io/example/example-devcontainer
+    push: true
 ```
 
-## Azure DevOps Task
```

Comment:
Is there a reason we don't want a dedicated section on the Azure DevOps Task?

Reply:
This shows up in the actual README in the GitHub Marketplace for the Action which looks really strange.

See https://github.com/marketplace/actions/devcontainers-ci

The content for the AzDO Task is in a completely different spot in the repo, so the top-level README does not apply to it. See https://marketplace.visualstudio.com/items?itemName=devcontainers.ci

Reply:
Oh got it, this helps me understand what you were describing in the original PR message. Thanks!